### PR TITLE
Run Docker builds on large workers, skip asset compression

### LINKED ISSUE #2487: Move docker image build to large workers
BuildKit is already doing a ton of stuff in parallel so it's easy to hang a standard worker.

### PR DESCRIPTION
**Pull request by kberzinch**
Closes https://github.com/RoboJackets/apiary/issues/2487

### CODE REVIEW

**Review by JoshOldenburg — DISMISSED** [review]

File: pipelines/information-technology/apiary.yml [hunk 1]
```diff
@@ -734,8 +744,11 @@ jobs:
           params:
             BUILDKIT_SECRET_composer_auth: composer-auth/auth.json
             BUILDKIT_PROGRESS: plain
+            TARGET: backend-uncompressed
```
Comment (evan10s) [inline pipelines/information-technology/apiary.yml:747]: Not sure where the other Docker build command lives, but for clarity, you should probably explicitly set the target for that as well

Comment (kberzinch) [reply pipelines/information-technology/apiary.yml:747]: It's in `job: build-main`, will update

Comment (kberzinch) [reply pipelines/information-technology/apiary.yml:748]: Added

File: pipelines/information-technology/apiary.yml [hunk 2]
```diff
@@ -407,7 +408,7 @@ jobs:
         args:
         - -i
         - >-
-            /job/c\job "apiary-test" {
+            /job "apiary" {/c\job "apiary-test" {
```
Comment (evan10s) [inline pipelines/information-technology/apiary.yml:411]: What is this for?

**Review by evan10s — APPROVED** [review]

File: pipelines/information-technology/apiary.yml (hunk 2 above)
Comment (kberzinch) [reply pipelines/information-technology/apiary.yml:411]: https://github.com/RoboJackets/apiary/pull/2359#discussion_r864374987

**Pull request merged**